.typos.toml

```diff
@@ -0,0 +1,28 @@
+# Configuration for https://github.com/crate-ci/typos
+
+[files]
+extend-exclude = [
+    "resources/linux-config-*",
+    "hypervisor/src/kvm/x86_64/mod.rs",
+]
+
+[default.extend-words]
+# External Interrupt
+EXTINT = "EXTINT"
+extint = "extint"
+INOUT = "INOUT"
+ba = "ba"
+Cros = "Cros" # CrosVM
```

Comment:
The official [crosvm](https://crosvm.dev/book/running_crosvm/index.html) documentation seems to consistently spell it "crosvm", all lowercase, so maybe we should follow suit and not have this exception?

Reply:
The typos utility is a little funny sometimes. It only complains about "Cros" in "CrossVM" and doesn't recognize the world as a whole.